lib/cisco_node_utils/node.rb

```diff
@@ -82,8 +82,8 @@ def config_get(feature, name, *args)
         # Just get the whole output
         return massage(show(ref.config_get, :structured), ref)
       elsif token[0].kind_of?(Regexp)
-        return massage(find_ascii(show(ref.config_get, :ascii),
-                                  token[-1], *token[0..-2]), ref)
+        return massage(Cisco.find_ascii(show(ref.config_get, :ascii),
+                                        token[-1], *token[0..-2]), ref)
```

Comment:
Context:  This code was originally added in [this commit](https://github.com/cisco/cisco-network-node-utils/commit/dc9e1141d1cb2ec9974adc7625c02da46db7134e).  It was removed in [this commit](https://github.com/cisco/cisco-network-node-utils/commit/b31d506c84068f716b17c1c17fc81ec4f82bd12a#diff-73ac5adb236895c412f98beb3210f365L103).